### PR TITLE
Cherry picking fun stuff from upstream

### PR DESCRIPTION
**Pull request by Michiyamenotehifunana**
## About The Pull Request

This PR ports the following PRs from upstream:
https://github.com/discordia-space/CEV-Eris/pull/5757
https://github.com/discordia-space/CEV-Eris/pull/5703
https://github.com/discordia-space/CEV-Eris/pull/5705
https://github.com/discordia-space/CEV-Eris/pull/5700
https://github.com/discordia-space/CEV-Eris/pull/5775
https://github.com/discordia-space/CEV-Eris/pull/5704
https://github.com/discordia-space/CEV-Eris/pull/5724
https://github.com/discordia-space/CEV-Eris/pull/5693
https://github.com/discordia-space/CEV-Eris/pull/5748
https://github.com/discordia-space/CEV-Eris/pull/5756

Due to lack of Erismed2 (as of this PR being made), 5729 from upstream is not included here. It would have heavily nerfed robotic limbs.

Most prominently:
- Removing Excelsior Implants now provide feedback
- Breathmasks get an action button
- Glue and Bluespace Polish are now a thing
- Armor vests with shoulder and kneepads, that can spawn in maint (and I think operatives spawn with them by default now)
- New sprites for stock parts
- Bones are now slightly less liable to break
- Some armors have been rebalanced

## Changelog
```changelog Toriate
add: Ported a bunch of fun stuff from upstream
```

### CODE REVIEW

File: code/game/objects/items/weapons/implant/implants/excelsior.dm [hunk 1]
```diff
@@ -99,10 +99,13 @@
 	for(var/datum/antagonist/A in wearer.mind.antagonist)
 		if(A.id == antag_id)
 			A.remove_antagonist()
-
+	wearer.visible_message(SPAN_DANGER("As \the [src] is removed from \the [wearer]..."))
 	if(prob(66))
+		wearer.visible_message(SPAN_DANGER("\The [wearer]'s [part.name] violently explodes from within!"))
```
Comment (Jamini) [inline code/game/objects/items/weapons/implant/implants/excelsior.dm:104]: What.


No.

Emojicracy this.

**Review by Jamini — CHANGES_REQUESTED** [review]
Please get an emojicracy on the excel implant change. Since it makes removing the implant have a 2/3 chance of gibbing the implanted limb and killing the user via brainloss.

This seems like something we would almost immediately want to modify because we don't value the same bullshit as Eris proper.

File: code/game/objects/items/weapons/implant/implants/excelsior.dm (hunk 1 above)
Comment (cebutris) [reply code/game/objects/items/weapons/implant/implants/excelsior.dm:104]: This doesn't actually change functionality. The code works exactly the same as it does currently. The only difference is now there's flavor text.

Comment (Jamini) [reply code/game/objects/items/weapons/implant/implants/excelsior.dm:104]: Hmm...alright. Fine.

**Review by Jamini — APPROVED** [review]
Everything seems in order.

**Pull request merged**